dataservice/api/sequencing_experiment/models.py

```diff
@@ -44,8 +44,6 @@ class SequencingExperiment(db.Model, Base):
     mean_depth = db.Column(db.Float())
     total_reads = db.Column(db.Integer())
     mean_read_length = db.Column(db.Float())
-    aliquot_id = db.Column(KfId(),
-                           db.ForeignKey('aliquot.kf_id'), nullable=False)
     genomic_files = db.relationship(GenomicFile,
                                     cascade="all, delete-orphan",
                                     backref=db.backref(
```

Comment:
The name of the backref should be `sequencing_experiment`.

Reply:
👍